mlir/test/e2e/GemmScaled.toml

```
@@ -0,0 +1,56 @@
+directory = "GemmScaled"
+prefix = "rocmlir-gen"
+suffix = "--operation gemm -out_dtype f32 --arch %arch %pv %random_data %rocmlir_gen_flags | rocmlir-driver -c | mlir-runner -O2 --shared-libs=%linalg_test_lib_dir/libmlir_rocm_runtime%shlibext,%conv_validation_wrapper_library_dir/libconv-validation-wrappers%shlibext,%linalg_test_lib_dir/libmlir_runner_utils%shlibext,%linalg_test_lib_dir/libmlir_float16_utils%shlibext --entry-point-result=void | FileCheck %s --check-prefix="
+
+[[axis]]
+name = "transA"
+values = ["true", "false"]
+prefix = "--transA="
+
+[[axis]]
+name = "transB"
+values = ["true", "false"]
+prefix = "--transB="
+
+[[axis]]
+name = "transC"
+values = ["true", "false"]
+prefix = "--transC="
+
+[[axis]]
+name = "data type"
+values = ["f4E2M1FN"]
+prefix = "-t "
+
+[[axis]]
+name = "scaledGemm"
+values = ["true", "false"]
```

do we want scaledGemm false here?

---

also, what does it mean scaledGemm=false, then -scale_a_dtype=f8E8M0FNU -scale_b_dtype=f8E8M0FNU?

---

This is also testing regular F4 gemms without scales  if `--scaledGemm = false` . If it is false then `-scale_a_dtype` and `scale_b_dtype` will be ignored.

---

that's confusing, I think we should throw an error if scaledGemm is true and the user still passes -scale_a_dtype and -scale_b_dtype, that sounds like a user error?